app/services/api/v1/target_service.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
+    class TargetService
+      def initialize(target_params, current_user, longitude, latitude)
+        @target_params = target_params
+        @current_user = current_user
+        @longitude = longitude
+        @latitude = latitude
+      end
+
+      def create
+        location = ActiveRecord::Point.new(@longitude, @latitude)
+        @current_user.targets.create!(@target_params.merge(location:))
+      end
```

Comment:
don't we need to notified matched users when we create a Target? Maybe you should start adding a link to the ticket on the PR description, to reviewers have more context of what the feature should do, wdyt?

Reply:
I send a request to access your Trello to see the ticket :)

Reply:
The notification system is addressed in another [Ticket](https://trello.com/c/DwVWDJIY/8-5-as-a-user-i-should-receive-notifications-when-a-compatible-target-has-been-created-so-we-can-start-getting-connected).

And im not sure where i can find the request to access the Trello, but i believe you can join using [this](https://trello.com/invite/b/gFx4RBmP/ATTI27dde06fb208a480a9906800d0778f73FCCEC717/target-mvd-api-rodrigo-miranda) link. Let me know if it works!